### PR TITLE
OTE landing page

### PR DESCRIPTION
**Pull request by vyashina**

### CODE REVIEW

File: OTE_landing_page.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# OpenVINO™ Training Extensions {#ote_documentation}
+[![python](https://img.shields.io/badge/python-3.8%2B-green)]()
+[![black](https://img.shields.io/badge/code%20style-black-000000.svg)]()
+[![mypy](https://img.shields.io/badge/%20type_checker-mypy-%231674b1?style=flat)]()
+[![openvino](https://img.shields.io/badge/openvino-2021.4-purple)]()
+
+OpenVINO™ Training Extensions (OTE) provide a suite of advanced algorithms to train
+Deep Learning models and convert them using the [OpenVINO™
+toolkit](https://software.intel.com/en-us/openvino-toolkit) for optimized
+inference. It allows to export and convert the models to the needed format. OTE independently create and train the model. It is open-sources and available on [GitHub](https://github.com/openvinotoolkit/training_extensions).
```
Comment (Ilya-Krylov) [inline OTE_landing_page.md]: It allows you to export and convert
or
It allows exporting and converting

Comment (Ilya-Krylov) [inline OTE_landing_page.md]:
```suggestion
inference. It allows to export and convert the models to the needed format. OTE independently create and train the model. It is open-sourced and available on [GitHub](https://github.com/openvinotoolkit/training_extensions).
```

File: OTE_landing_page.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+## Detailed Workflow
+![](training_extensions_framework.png)
+
+1. To start working with OTE, prepare and annotate your dataset. For example, on CVAT.
+
+2. OTE train the model, using training interface, and evaluate the model quality on your dataset, using evaluation and inference interfaces.
+
+Note: prepare the saparate dataset or split the dataset you have for the more accurate quality evaluation.
```
Comment (Ilya-Krylov) [inline OTE_landing_page.md]:
```suggestion
Note: prepare a separate dataset or split the dataset you have for the more accurate quality evaluation.
```

Comment (Ilya-Krylov) [inline OTE_landing_page.md]:
```suggestion
Note: prepare the saparate dataset or split the dataset you have for more accurate quality evaluation.
```

File: OTE_landing_page.md [hunk 3]
```diff
@@ -0,0 +1,80 @@
+# OpenVINO™ Training Extensions {#ote_documentation}
```
Comment (Ilya-Krylov) [inline OTE_landing_page.md:1]: `{#ote_documentation}` what is this for?

**Review by Ilya-Krylov — CHANGES_REQUESTED** [review]

**Comment by Ilya-Krylov** [comment]
Is there an intention to replace existing readme.md file?

File: OTE_landing_page.md (hunk 3 above)
Comment (vyashina) [reply OTE_landing_page.md:1]: it's sphinx tag to build the doc on our website

File: OTE_landing_page.md [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+inference. It allows to export and convert the models to the needed format. OTE independently create and train the model. It is open-sourced and available on [GitHub](https://github.com/openvinotoolkit/training_extensions).
+
+## Detailed Workflow
+![](training_extensions_framework.png)
+
+1. To start working with OTE, prepare and annotate your dataset. For example, on CVAT.
+
+2. OTE train the model, using training interface, and evaluate the model quality on your dataset, using evaluation and inference interfaces.
+
+Note: prepare a saparate dataset or split the dataset you have for more accurate quality evaluation.
```
Comment (AnastasiaKazantaeva) [inline OTE_landing_page.md]: separate

**Review by Ilya-Krylov — DISMISSED** [review]

**Review by AnastasiaKazantaeva — DISMISSED** [review]

**Pull request merged**